server/container_create.go

```diff
@@ -373,3 +383,19 @@ func (s *Server) generateContainerIDandName(podName string, name string, attempt
 	}
 	return id, name, err
 }
+
+// getAppArmorProfileName gets the profile name for the given container.
+func (s *Server) getAppArmorProfileName(annotations map[string]string, ctrName string) string {
```

Comment:
This doesn't make any sense. What is the point of a "default profile" if we're not automatically enforcing it for all containers.